sdk/php/template/README.md

```diff
@@ -0,0 +1,194 @@
+# Dagger PHP SDK #
+
+You will require the Dagger engine setup locally, and it is currently up and running.
+
+## Quickstart ##
+
+### Initialise the module ###
+
+To initialise the module, call the following:
+
+```
+dagger init --sdk="sdk/php" <path-to-module>
+```
+
+This will create a `dagger.json` configuration file.
+
+The `path-to-module` argument can be omitted and the module will be initialised where the command was called from.
+
+### Setup The Development Environment ###
+
+To setup the development environment, call the following:
+
+```
+dagger develop --mod=<path-to-module>
+```
+
+This will generate a copy of the classes available to you from the SDK. Making it easier to see what you have to work with. It will also generate a `src/` directory, this is required. This is where Dagger will search for available functions.
+
+The `--mod` option must be a git url, or relative file path. The argument may be omitted if the command is called within the directory containing your module's `dagger.json` 
+
+### Check Module Functions ###
+
+You can find out what methods are available by calling the following:
+
+```
+dagger functions --mod=<path-to-module>
+```
+
+The `--mod` option must be a git url, or relative file path. The argument may be omitted if the command is called within the directory containing your module's `dagger.json` 
+
+You should find two functions:
+- echo
+- grep-dir
+
+These functions are written within `src/Example.php`.  
+This aptly named file demonstrates how DaggerFunctions should be written.
+
+
+### Call Module Functions ###
+
+```text
+dagger call echo  --value="hello world" -mod=<path-to-your-module>
```

Comment:
Nit: The `-m` flag is best at the beginning of the command. I usually put it right after `dagger`, but at the very least it should be on the `dagger <top-level>` command:
```suggestion
dagger call -m <path-to-your-module> echo  --value="hello world" 
```

Reply:
That is clearer; I've adjusted it to match your suggestion.